server/app.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route("/")
+def index():
+    return "Hello World!"
```

Comment:
This seems to me that the app isnt ready yet...

Reply:
This will become the CMS, but the current app is using hard coded content

Reply:
So it is ready